style/js/util.js

```diff
@@ -5,10 +5,19 @@ function restrictLayer(def, layer) {
   return filteredClone(def, ["==", "layer", layer], "_layer_" + layer);
 }
 
+function cp(obj) {
+  return JSON.parse(JSON.stringify(obj));
```

Comment:
For reference:

The lodash method `cloneDeep`: https://lodash.com/docs/4.17.15#cloneDeep

Coming soon to browsers near you: `structuredClone`: https://www.chromestatus.com/feature/5630001077551104

Reply:
Based on discussion on Slack, pulling in a 3rd party library just for the clone function is probably not worth the squeeze, but we should use a library clone function if we pull in one of these libraries for some other reason in the future.